.cirrus.yml

```diff
@@ -1,58 +1,49 @@
-env:
-  PACT_BROKER_FEATURES: publish_pacts_using_old_api
 
-BUILD_TEST_TASK_TEMPLATE: &BUILD_TEST_TASK_TEMPLATE
-  arch_check_script:
-    - uname -am
-  test_script:
-    - node --version
-    - script/ci/build-and-test.sh
+INSTALL_GH_CLI: &INSTALL_GH_CLI
+  install_gh_cli_script: |
+    curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg \
+    && chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg \
+    && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | tee /etc/apt/sources.list.d/github-cli.list > /dev/null \
+    && apt update \
+    && apt install gh -y
 
-# These are probably expected to fail in the post install script
-# until we are packing v2.0.0 of pact-ruby-standalone that supports
-# arm64 linux - as per https://github.com/pact-foundation/pact-js-core/issues/416
-# Error: Error while locating pact binary: Cannot find binary for platform 'linux' with architecture 'arm64'.
-# linux_arm64_task:
-#   skip: "changesInclude('.github/**')" 
-#   env:
-#     matrix:
-#       - IMAGE: node:16-slim
-#       - IMAGE: node:18-slim
-#       - IMAGE: node:20-slim
-#   arm_container:
-#     image: $IMAGE
-#   install_script:
-#     - apt update --yes && apt install --yes curl python3 make build-essential g++ unzip zip
-#   << : *BUILD_TEST_TASK_TEMPLATE
-
-linux_amd64_task:
+RELEASE: &RELEASE
+  release_script: ./script/ci/release.sh
   env:
-    matrix:
-      - IMAGE: node:16-slim
-      - IMAGE: node:18-slim
-      - IMAGE: node:20-slim
-  container:
-    image: $IMAGE
-  install_script:
-    - apt update --yes && apt install --yes curl python3 make build-essential g++ unzip zip
-  << : *BUILD_TEST_TASK_TEMPLATE
+    GITHUB_TOKEN: ENCRYPTED[636f316600928de28b5c36027cc39d9796bc0d0eca2a181368f255ad61540f13bb38cdce09b6428774f315bbf45e0ada]
```

Comment:
Note - Pact-Foundation Cirrus CI account will need a scoped `GITHUB_TOKEN`

Reply:
What is needed here sorry Saf - do we need to bake a new one with scoped permissions instead of (presumably) wider perms here?

Reply:
hiya,

so currently this is using a PAT under my own user scope, which has been attached to my pact js core fork in cirrus ci.

it creates an encrypted secret 

https://cirrus-ci.org/guide/writing-tasks/#encrypted-variables

which is usable at the repo level.

the token can only be encrypted by the associated repo and if the user triggering has suitable perms.

this will mean for merging 

1. will need a scoped PAT creating in pact foundation org, and added to our vault that Beth setup.
2. will need the scoped PAT linked in pact-js-core cirrus config ( under pact foundation org not my fork )

token is required perms to

- list releases
- upload assets to releases